www/nginx/src/opnsense/mvc/app/views/OPNsense/Nginx/index.volt

```diff
@@ -55,6 +58,89 @@
             });
         });
     }
+
+    function ngnx_show_conf() {
+
+        $("#nginx_conf tbody").empty().append('<tr><td class="placeholdertd">{{ lang._("Waiting for response..") }}</td></tr>');
+        $("#config_help_text").hide();
+        ajaxCall(url="/api/nginx/settings/showconfig/", sendData={}, callback=function(data,status) {
+            if (data['time'] && data['config']) {
+                var L = 0;
+                var content = [];
+                $.each(data['config'], function(index, line) {
+                    L = line.indexOf('# configuration file ') > -1 ? 0 : L + 1;
+                    content.push('<tr><td class="l-number">' + L.toString() + '</td><td class="config-line"><span>' + line + '</span></td></tr>');
+                });
+                $("#nginx_conf tbody").empty().append(content.join());
+                $("#config_help_text").show();
+                BootstrapDialog.show({
+                   type: BootstrapDialog.TYPE_INFO,
+                   title: "{{ lang._('NGINX config loaded successfully') }}",
+                   message: "{{ lang._('NGINX config loaded. Config file created at') }}" + ": " + (new Date(data['time']*1000)).toLocaleString(),
+                   buttons: [{
+                       label: 'Ok',
+                       action: function(dlg){
+                           dlg.close();
+                       }
+                   }]
+                });
+            } else {
+                  $("#nginx_conf").html("{{ lang._('Empty response from the backend. Please check logs.') }}");
+            }
+        });
+    }
+
+    function ngnx_test_conf() {
+        ajaxCall(url="/api/nginx/settings/testconfig/", sendData={}, callback=function(data,status) {
+            if (data['response'].indexOf('test failed') > -1) {
+                 BootstrapDialog.show({
+                    type: BootstrapDialog.TYPE_DANGER,
+                    title: "{{ lang._('NGINX config test failed') }}",
+                    message: data['response'],
+                    buttons: [{
+                        label: 'Ok',
+                        action: function(dlg){
+                             dlg.close();
+                        }
+                    }]
+                });
+            } else {
+                 BootstrapDialog.show({
+                    type: BootstrapDialog.TYPE_INFO,
+                    title: "{{ lang._('NGINX config test is successful') }}",
+                    message: "{{ lang._('NGINX config test is successful') }}",
+                    buttons: [{
+                        label: 'Ok',
+                        action: function(dlg){
+                             dlg.close();
+                        }
+                    }]
+                });
+            }
+        });
+    }
+
+    $( document ).ready(function() {
```

Comment:
@fichtner Hi! what do you think about using an arrow for this (since IE support has been dropped)? Which style is better to follow? (jquery docs still offer `$( handler )` as recommended but its just not jQuery-specific imho)